src/aim/utils/monitoring.py

```
@@ -0,0 +1,330 @@
+        uptime = time.time() - self.start_time
+        
+        # Basic system checks
+        details = {
+            "system": self._check_system_health(),
+            "application": self._check_application_health(),
+            "custom": self._run_custom_checks()
+        }
+        
+        # Determine overall status
+        status = "healthy"
+        for category, checks in details.items():
```

Ruff (0.12.2)

Loop control variable `category` not used within loop body

Rename unused `category` to `_category`

(B007)